src/pages/steps/AdditionalInformationStep/PriceInformation.tsx

```diff
@@ -212,7 +212,9 @@ const PriceInformation = ({
       replace(priceInfo.length ? priceInfo : defaultPriceInfoValues.rates);
 
       return onValidationChange(
-        hasUitpasLabel ? ValidationStatus.WARNING : ValidationStatus.NONE,
+        hasUitpasLabel && scope === OfferTypes.EVENTS
+          ? ValidationStatus.WARNING
+          : ValidationStatus.NONE,
```

Comment:
```suggestion
        hasUitpasLabel
          ? ValidationStatus.WARNING
          : ValidationStatus.NONE,
```

The condition `scope === OfferTypes.EVENTS` can be part of the hasUitpasLabel constant